lib/util/get-npmignore.js

```diff
@@ -90,7 +89,10 @@ function parseWhiteList(files) {
 
     for (const file of files) {
         if (typeof file === "string" && file) {
-            const body = file.replace(SLASH_AT_BEGIN_AND_END, "")
+            const body = path.posix
```

Comment:
It seems that [`ignore`](https://www.npmjs.com/package/ignore) only supports posix :eyes: